gtfs-realtime/spec/en/reference.md

```diff
@@ -502,11 +505,31 @@ An internationalized message containing per-language versions of a snippet of te
 
 A localized string mapped to a language.
 
+| _**Field Name**_ | _**Type**_ | _**Required**_ | _**Cardinality**_ | _**Description**_ |
+|------------------|------------|----------------|-------------------|-------------------|
+| **text** | [string](https://developers.google.com/protocol-buffers/docs/proto#scalar) | Required | One | A UTF-8 string containing the message. |
+| **language** | [string](https://developers.google.com/protocol-buffers/docs/proto#scalar) | Conditionally required | One | BCP-47 language code. Can be omitted if the language is unknown or if no internationalization is done at all for the feed. At most one translation is allowed to have an unspecified language tag - if there is more than one translation, the language must be provided. |
+
+## _message_ TranslatedImage
+
+An internationalized message containing per-language versions of an image. One of the images from a message will be picked up. The resolution proceeds as follows: If the UI language matches the language code of a translation, the first matching translation is picked. If a default UI language (e.g., English) matches the language code of a translation, the first matching translation is picked. If some translation has an unspecified language code, that translation is picked.
```

Comment:
I completely understand the intent of this sentence, but I think the wording is not correct. This sounds line a consumer's api documentation, while we mainly are talking about how to produce these files. So all the "is picked" should be changed IMHO. Maybe to something that makes it as a recommendation to the consumers. Or maybe a bit stricter than that... like "should be picked"

Reply:
This is text copied + adapted from TranslatedString. I'm for clarifying this but maybe that can be done in an other PR and fix both at the same time?